metadata-integration/java/spark-lineage/build.gradle

```diff
@@ -95,17 +95,23 @@ dependencies {
   provided(externalDependency.sparkSql)
   provided(externalDependency.sparkHive)
 
+  implementation(externalDependency.sparkBigQuery){
```

Comment:
can we make it `compileOnly` instead of `implementation` ? 
that way these jars won't show up in the shadow jar. 
whoever is using spark, if they are using bigquery, they will have those jars on the runtime classpath anyway.

Reply:
If we make dependency compleOnly then for the cases, where bigquerry is not needed, bigquerry jars need to be added otherwise we get runtime classnot found exception.